<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>newColor</class>
 <widget class="QWidget" name="newColor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>320</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(90, 90, 90);</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>2</number>
     </property>
     <item>
      <widget class="QLabel" name="label_10">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>25</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>黑体</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(152, 152, 152);
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>R</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="rightMargin">
        <number>4</number>
       </property>
       <item>
        <widget class="QLabel" name="label">
         <property name="minimumSize">
          <size>
           <width>65</width>
           <height>45</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>65</width>
           <height>45</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>黑体</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>原值:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>45</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>45</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>黑体</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="minimumSize">
          <size>
           <width>65</width>
           <height>45</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>65</width>
           <height>45</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>黑体</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>新值:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit">
         <property name="minimumSize">
          <size>
           <width>90</width>
           <height>35</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>35</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Adobe 黑体 Std R</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(152, 152, 152);
color: rgb(255, 255, 255);</string>
         </property>
         <property name="frame">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="label_11">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>25</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>黑体</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(152, 152, 152);
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>G</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="rightMargin">
        <number>4</number>
       </property>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="minimumSize">
          <size>
           <width>65</width>
           <height>45</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>65</width>
           <height>45</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>黑体</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>原值:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>45</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>45</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>黑体</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_6">
         <property name="minimumSize">
          <size>
           <width>65</width>
           <height>45</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>65</width>
           <height>45</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>黑体</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>新值:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_2">
         <property name="minimumSize">
          <size>
           <width>90</width>
           <height>35</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>35</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Adobe 黑体 Std R</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(152, 152, 152);
color: rgb(255, 255, 255);</string>
         </property>
         <property name="frame">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="label_12">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>25</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>黑体</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(152, 152, 152);
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>B</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <property name="rightMargin">
        <number>4</number>
       </property>
       <item>
        <widget class="QLabel" name="label_7">
         <property name="minimumSize">
          <size>
           <width>65</width>
           <height>45</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>65</width>
           <height>45</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>黑体</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>原值:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_8">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>45</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>45</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>黑体</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_9">
         <property name="minimumSize">
          <size>
           <width>65</width>
           <height>45</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>65</width>
           <height>45</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>黑体</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>新值:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_3">
         <property name="minimumSize">
          <size>
           <width>90</width>
           <height>35</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>35</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Adobe 黑体 Std R</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(152, 152, 152);
color: rgb(255, 255, 255);</string>
         </property>
         <property name="frame">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <property name="topMargin">
        <number>8</number>
       </property>
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>35</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>黑体</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>确定</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_2">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>35</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>黑体</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>清除</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
